lib/handlers/quote/quote.ts

```diff
@@ -281,8 +281,13 @@ export class QuoteHandler extends APIGLambdaHandler<
             currencyOut.symbol
           }. Chain: ${chainId}`
         )
-
-        swapRoute = await router.route(amount, currencyOut, TradeType.EXACT_INPUT, swapParams, routingConfig)
+        try {
+          swapRoute = await router.route(amount, currencyOut, TradeType.EXACT_INPUT, swapParams, routingConfig)
+        } catch (error) {
+          metric.putMetric(`GET_QUOTE_500_CHAINID: ${chainId}`, 1, MetricLoggerUnit.Count)
```

Comment:
For the same reason as alarming, do we want to export 5xx metrics for CHAINS_NOT_MONITORED, aka testnets?

Reply:
Let's create a ticket for that

Reply:
oh :facepalm: the code is already in place lol

Reply:
i think it still makes sense to have the metric here, just to see whats going in, so the default is visibility